static/js/components/material/Button.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import React from 'react';
+
+export default class Button extends React.Component {
+  render() {
+    const { children, className, ...otherProps } = this.props;
+
+    return <button
+      className={className ? `mdc-button ${className}` : 'mdc-button'}
```

Comment:
Nit: I think we use the `{'mdc-button ${className ? className : ""}'` pattern in other projects, which to me reads easier.